BUILD_OSX.md

```diff
@@ -17,7 +17,9 @@ Execute the `Update Shell Profile.command` script that is provided with the inst
 
 ### OSX SDK
 
-You will need the OSX SDK for building. The easiest way to get this is to install Xcode from the App Store.
+You will need version `10.12` of the OSX SDK for building. You can get that from [here](https://github.com/phracker/MacOSX-SDKs). You must copy it in to your Xcode SDK directory, e.g.
```

Comment:
Perhaps it makes sense to add here a short explanation why newer sdks fail and with what error, so the issue is searchable by those who try to build it and see the runtime crashes.